components/organisms/ContributorCard/contributor-card.tsx

```diff
@@ -50,42 +50,44 @@ const ContributorCard = ({ className, contributor, topic, repositories, range, s
     <Card className={className && className}>
       <div className="flex flex-col gap-3">
         <div className="flex items-center justify-between w-full gap-2">
-          <Link href={`/u/${username}`} as={`/u/${username}`}>
-            <div className="flex items-center gap-2">
+          <div className="flex items-center gap-2">
+            <Link href={`/u/${username}`} as={`/u/${username}`}>
```

Comment:
TLDR;, the link was wrapping too much so clicking the OSCR login button also clicked the link which caused the user to navigate to the profile page associated to the contributor card they clicked on.

Now the link for the user profile only wraps the avatar and the username.